src/main/java/com/googlecode/lanterna/gui2/ImageComponent.java

```diff
@@ -0,0 +1,54 @@
+package com.googlecode.lanterna.gui2;
+
+import com.googlecode.lanterna.*;
+import com.googlecode.lanterna.graphics.*;
+
+/**
+ * 
+ * @author ginkoblongata
+ */
+public class ImageComponent extends AbstractInteractableComponent {
+    
+    TextImage textImage;
```

Comment:
You probably meant this should be `private`

Reply:
Yep, changed to private.